blocks/log.py

```diff
@@ -64,10 +69,3 @@ def previous_row(self):
     @property
     def last_epoch_row(self):
         return self[self.status['_epoch_ends'][-1]]
-
-    def to_dataframe(self):
-        """Convert a log into a :class:`.DataFrame`."""
-        if not PANDAS_AVAILABLE:
-            raise ImportError("The pandas library is not found. You can"
-                              " install it with pip.")
-        return DataFrame.from_dict(self, orient='index')
```

Comment:
Well, that's why I added the snippet to the docs. I think there's certainly a benefit to having it if we start supporting multiple backends. However, there's also a cost, and I've seen many people run into installation issues with Blocks or wait a long time for Pandas to compile (especially on clusters), while you are the only person I know of who uses this one-liner. So I'd argue that the costs outweigh the tiny, tiny benefit of being able to type `log.to_dataframe()` instead of `DataFrame.from_dict(log, orient='index')`.

I'll check if `blocks-plot` is broken. Tests passed, but perhaps it's not covered by tests.